cmd/ztest/ztest.c

```diff
@@ -6005,136 +5980,6 @@ ztest_fault_inject(ztest_ds_t *zd, uint64_t id)
 	umem_free(pathrand, MAXPATHLEN);
 }
 
-/*
- * Verify that DDT repair works as expected.
- */
-void
-ztest_ddt_repair(ztest_ds_t *zd, uint64_t id)
-{
```

Comment:
With this test case removed I'm concerned that we no longer have any test coverage for existing pools which which did make use of `dedupditto` functionality.  We'll be risking accidentally breaking this functionality again.  As part of this removal why don't we add a basic test case to the ZTS which verifies that `dedupditto` repair continues to work as intended.  We can create a tiny custom test pool for the purpose as was done for some of the `zpool_upgrade` tests.